scripts/plot_dataframe.py

```diff
@@ -1,30 +1,23 @@
 #!/usr/bin/env python3
 
 import sys
+import logging
 import argparse
-import matplotlib  # Install PYQT5 manually if you want to test this helper function
-matplotlib.use("Qt5Agg")
```

Comment:
removing this line will stop the script using PyQt5 for rendering

Reply:
Yes, Tkinter is the defacto python graphical framework.